airbyte-integrations/connectors/source-sigma-computing/manifest.yaml

```diff
@@ -0,0 +1,909 @@
+version: 5.12.0
+
+type: DeclarativeSource
+
+description: >
+  This is the setup for the Sigma Computing source that ingests data from the
+  sigma API. 
+
+
+  Sigma is next-generation analytics and business intelligence that scales
+  billions of records using spreadsheets, SQL, Python, or AI—without
+  compromising speed and security https://www.sigmacomputing.com/
+
+
+  In order to use this source, you must first create an account on Sigma
+  Computing. Go to Account > General Settings and review the Site section for
+  the Cloud provider, this will be used to find the base url of your API.
+  Compare it at https://help.sigmacomputing.com/reference/get-started-sigma-api
+
+
+  Next, head over to Developer Access and click on create. This will generate
+  your Client ID and Client Secret required by the API. You can learn more about
+  the API here https://help.sigmacomputing.com/reference
+
+check:
+  type: CheckStream
+  stream_names:
+    - files
+
+definitions:
+  streams:
+    files:
+      type: DeclarativeStream
+      name: files
+      primary_key:
+        - id
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: /files
+          http_method: GET
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path:
+              - entries
+        paginator:
+          type: DefaultPaginator
+          page_token_option:
+            type: RequestOption
+            inject_into: request_parameter
+            field_name: page
+          page_size_option:
+            type: RequestOption
+            field_name: limit
+            inject_into: request_parameter
+          pagination_strategy:
+            type: OffsetIncrement
```

Comment:
I don't believe the offset is the correct pagination method for the API. https://help.sigmacomputing.com/reference/listdatasets It uses pages, can you double check it?

Reply:
You 
are correct. Updated :)